Service Connector  Hub/func.yaml

```diff
@@ -7,4 +7,5 @@ memory: 1024
 timeout: 120
 config:
   DATADOG_HOST: https://http-intake.logs.datadoghq.com/v1/input 
-  DATADOG_TOKEN: <DATADOG_API_TOKEN>
\ No newline at end of file
+  DATADOG_TOKEN: <DATADOG_API_TOKEN>
+  # DATADOG_TAGS: "prod:true"                                        # Tags associated with logs
```

Comment:
should this be commented out?

Reply:
Similar to the other one it's just a customer option. Meant to be commented out for customer to specify their own custom tags.